tsconfig.json

```diff
@@ -3,7 +3,6 @@
   "compilerOptions": {
     "module": "ES2020",
     "lib": ["ES2019", "DOM"],
-    "importsNotUsedAsValues": "error",
```

Comment:
for posterity, this is a typescript 5 change: https://stackoverflow.com/questions/75449286/how-to-fix-flag-importsnotusedasvalues-is-deprecated-and-will-stop-functionin

see also addition of the following line below:
`"verbatimModuleSyntax": true`